test/client/requestUtil.js

```diff
@@ -175,12 +178,102 @@ test('client RequestUtil', (t) => {
               t.deepEquals(s3Record.deviceId, record.deviceId, `${t.name}: deviceId`)
               t.deepEquals(s3Record.objectId, record.objectId, `${t.name}: objectId`)
               t.deepEquals(s3Record.historySite, record.historySite, `${t.name}: historySite.location`)
-              testCanDeleteHistorySites(t)
```

Comment:
why was this moved into a callback? seems that the two tests should be independent

Reply:
it is called after `testCanDeleteBookmarks`